methods/incidentReports.coffee

```diff
@@ -35,18 +47,20 @@ Meteor.methods
       Meteor.call("editUserEventLastIncidentDate", incident.userEventId)
     return incident._id
 
-  editIncidentReport: (incident) ->
+  editIncidentReport: (incident, userEventId) ->
```

Comment:
I think we do need to pass the userEventId...I just forgot to when the methods are called. Mostly because we no longer store the `userEventId` on the incidents. So without the parameter, we'd have to query for the event's ID in order to update the event metadata?